testscommon/dataRetriever/proofsPoolStub.go

```diff
@@ -0,0 +1,55 @@
+package dataRetriever
+
+import (
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+)
+
+// ProofsPoolStub -
+type ProofsPoolStub struct {
```

Comment:
rename to mock.. file as well.. can use scripts/generators/mockGenerator.sh

Reply:
renamed